cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/LocalizationUtils.java

```diff
@@ -42,6 +48,43 @@
 public final class LocalizationUtils {
   private static final Logger LOG = LoggerFactory.getLogger(LocalizationUtils.class);
 
+  public static List<File> getExtraJars(CConfiguration cConf) {
+    String extraJarConf = cConf.get(Constants.AppFabric.EXTRA_JARS);
```

Comment:
That is a `getStrings` method in cconf. You don't have to do the split yourself.